Source/macOS/OIDRedirectHTTPHandler.h

```diff
@@ -45,6 +45,18 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (instancetype)initWithSuccessURL:(nullable NSURL *)successURL;
 
+/*! @brief Starts listening on the loopback interface on a specified port, and returns a URL
+ with the base address. Use the returned redirect URI to build a @c OIDExternalUserAgentRequest,
```

Comment:
The indentation of this comment block is off, can you match the style used elsewhere?